.github/workflows/deploy_tre.yml

```diff
@@ -22,7 +22,9 @@ jobs:
     uses: ./.github/workflows/deploy_tre_reusable.yml
     with:
       ciGitRef: ${{ github.ref }}
-      e2eTestsCustomSelector: "extended or shared_services or extended_aad"
+      e2eTestsCustomSelector: >-
+        ${{ (github.event_name == 'schedule' && 'extended or shared_services or extended_aad') ||
+        (github.event_name == 'push' && 'extended or extended_aad') }}
```

Comment:
There's another event for this workflow which is the dispatch. I think you should add an "else" at the end to catch every other even with a default...